src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -26,26 +26,37 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
 }
 
 .Button {
-  @include button-base;
   @include focus-ring;
```

Comment:
We're not getting focus rings on the button anymore. That seems intentional since it's a div and not tabbable so I think some styles can be removed from here and the .tsx